### PR TITLE
Suggestions for the Spring 2016 policy review cycle

### PR DESCRIPTION
**Pull request by dehnert**

### CODE REVIEW

File: policies.tex [hunk 1]
```diff
@@ -47,7 +47,7 @@ \subsection{}After electing all other candidates, Members-at-Large will be elect
 \subarticle{Appointments}
 
 \subsection{}The Executive Committee may appoint additional officers, including but not limited to those required by the constitution or standing policies.
-\subsection{}The term of an appointed officer shall not extend beyond the end of the term of the appointing EC.
+\subsection{}The term of an appointed officer shall not extend more than one month beyond the end of the term of the appointing EC, and shall lapse upon their replacement taking office.
```
Comment (kchen) [inline policies.tex]: This should be "and shall lapse upon their replacement's taking office", although that reads weirdly.  Perhaps reword to "and shall lapse when their replacement takes office."

Although regardless of the wording above, the two parts of the sentence sound somewhat contradictory.  Perhaps "The term of an appointed officer shall lapse when their replacement takes office, no later than one month beyond the end of the term of the original appointing EC."

Still not quite happy with that text.

File: policies.tex [hunk 2]
```diff
@@ -73,7 +73,8 @@ \subsection{}Club meetings may be called by the EC, or by petition of five votin
...
 \section{}The Class Coordinator, after notifying the EC, may remove people from the class if they are interfering with the class or preventing other class members from learning.
 \section{}The EC must approve the list of class graduates. The Class Coordinator shall provide the EC with a list of class members, and provide a recommendation of which should graduate, which should not, and which deserve discussion.
-\section{}A class member must dance club level or be expected to dance club level soon to graduate. 
+\section{}A class member must dance club level or be expected to dance club level soon to graduate.
+\section{}Running the class often requires missing the first and second club tips (due to walkthroughs and class meeting, respectively). In recognition of this, the class coordinator and up to three class assistants they designate shall receive a free subscription for the terms the class is in session. The class coordinator must inform the EC who should receive free subscriptions. Each person receiving a free subscription must be missing at least half of club tips due to their service on the class committee.
```
Comment (kchen) [inline policies.tex]: s/a free subscription/free subscriptions/
That sentence also seems like it should somehow be combined with the following one.

File: policies.tex (hunk 1 above)
Comment (vboyce) [inline policies.tex]: Another wording possibility, parallel to the "Officers shall take office on the earlier of one week after the spring class ends or June 1" in the constitution, is "The term of an appointed officer shall not extend past their replacement taking office or one month after the new EC takes office."

**Comment by dehnert** [comment]
Here's a PDF of the (combined) [revised policies](https://github.com/tech-squares/govdocs/files/197008/policies.diff.pdf), with the new text in blue. I haven't yet made Kevin and Veronica's wording changes.

**Comment by dehnert** [comment]
Here's the [revised policies](https://github.com/tech-squares/govdocs/files/200720/policies.pdf) and the [changes highlighted](https://github.com/tech-squares/govdocs/files/200719/policies.diff.pdf).

**Comment by dehnert** [comment]
Here's [re-revised policies](https://github.com/tech-squares/govdocs/files/214403/policies.pdf) and the [changes highlighted](https://github.com/tech-squares/govdocs/files/214401/policies.diff.pdf). This now resolves #3 and #4.

**Comment by dehnert** [comment]
[Policies approved by the EC](https://github.com/tech-squares/govdocs/files/222069/policies.pdf) and [with changes highlighted](https://github.com/tech-squares/govdocs/files/222070/policies.diff.pdf). #3 is no longer resolved, but #4 still is.

**Pull request merged**